lib/steps.js

```diff
@@ -1131,7 +1131,7 @@ After(async function(scenario) {
       if ( !!scope.disable_error_screenshot ) {
         scope.addToReport(scope, {
           type: `row`,
-          value: `For security, ALKiln will avoid creating a screenshot for this error. It's possible a secret is being used on this screen.`
+          value: `For security, ALKiln will avoid creating a picture for this error. It's possible a secret is being used on this screen.`
```

Comment:
To confirm my understanding: did you change "picture" to "screenshot" because the files now have "picture" in the filename and not "screenshot"?

I'm down with adding an option to say "and I take a pic", but conflicted because "screenshot" really effectively communicated what exactly we're avoiding doing, and saying picture in all of the user-shown text is much more vague. "Picture of the page" is better, but a little wordier, and you haven't used it everywhere (not sure if that's intentional).

I'm hoping that consistency between the user-shown text and the filenames isn't as important in this case, and that folks who run on GitHub will know that the *.jpg files are screenshots, but idk if that's accurate.

Reply:
Sorry, should have read everything before responding to the top post. I'm currently more concerned about a mismatch in language. I've seen folks get really confused about stuff like that. I think elsewhere I changed it to "picture of the page", as you said, and intended to be consistent. Maybe sometimes use one and sometimes another? I dunno. May be worth asking an outside consultant.

Reply:
> I think elsewhere I changed it to "picture of the page", as you said, and intended to be consistent

That's probably good enough. It is pretty minor, maybe I just have an emotional attachment to screenshot for some reason 📷